keras/export/export_lib.py

```diff
@@ -453,6 +460,24 @@ def _spec_to_poly_shape(self, spec):
         spec_shape = str(spec_shape).replace("None", "b")
         return spec_shape
 
+    def _check_device_compatible(self):
+        from jax import default_backend as jax_device
+
+        if (
+            jax_device() == "gpu"
+            and len(tf.config.list_physical_devices("GPU")) == 0
+        ):
+            logging.warning(
+                "JAX backend is using GPU for export, but installed "
+                "TF package cannot access GPU. To use JAX-native serialization "
```

Comment:
Add some more info, e.g. ", so reloading the model with the TF runtime in the same environment will not work. "

Reply:
Updated, thanks!